### PR TITLE
[12.0] Migration of account_invoice_line_description

### PR DESCRIPTION
**Pull request by tafaRU**

### CODE REVIEW

**Comment by tafaRU** [comment]
Please note I didn't squash the commits generated by Transifex as they have conflicts.

File: account_invoice_line_description/readme/CONFIGURE.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+The user needs the check *Use only product description on invoice lines* setting on the Invoicing Settings.
```
Comment (rven) [inline account_invoice_line_description/readme/CONFIGURE.rst]: This setting needs to be checked on the Access Rights (Technical Settings) of the user

**Review by rven — COMMENTED** [review]
Tested the module on Runbot and it is working as expected!

File: account_invoice_line_description/readme/CONFIGURE.rst (hunk 1 above)
Comment (tafaRU) [reply account_invoice_line_description/readme/CONFIGURE.rst]: Do you mean in alternative?

Comment (rven) [reply account_invoice_line_description/readme/CONFIGURE.rst]: As discussed, there is a global setting in the accounting or you can enable it per user

**Comment by tafaRU** [comment]
@rven, modified `configuration ` part as discussed and rebased. Please check.

**Review by rven — APPROVED** [review]

**Review by ThomasBinsfeld — APPROVED** [review]

**Review by mpanarin — APPROVED** [review]

**Comment by tafaRU** [comment]
:green_heart:

File: account_invoice_line_description/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    'name': 'Account invoice line description',
+    'version': '12.0.1.0.0',
+    'category': 'Generic Modules/Accounting',
+    'author':   "Agile Business Group, "
+                "Tecnativa, "
+                "Odoo Community Association (OCA)",
+    'website': 'http://www.agilebg.com',
```
Comment (bodedra) [inline account_invoice_line_description/__manifest__.py]: Shouldn't be https://github.com/OCA/account-invoicing ?

**Review by bodedra — APPROVED** [review]

File: account_invoice_line_description/__manifest__.py (hunk 2 above)
Comment (tafaRU) [reply account_invoice_line_description/__manifest__.py]: done at https://github.com/OCA/account-invoicing/pull/429/commits/6b8862d39940ba31dbba97270d35ec8629029de7 Thanks!

**Comment by tafaRU** [comment]
@OCA/accounting-maintainers, could you please merge it?

**Comment by tafaRU** [comment]
thanks @pedrobaeza :metal:

**Comment by pedrobaeza** [comment]
Let's rock with v12!

**Pull request merged**